cabal-install/Distribution/Client/CmdConfigure.hs

```diff
@@ -83,7 +86,13 @@ configureAction (configFlags, configExFlags, installFlags, haddockFlags)
     baseCtx <- establishProjectBaseContext verbosity cliConfig
 
     -- Write out the @cabal.project.local@ so it gets picked up by the
-    -- planning phase.
+    -- planning phase. If old config exists, then print the contents
+    -- before overwriting
+    exists <- doesFileExist "cabal.project.local"
+    when exists $ do
+        putStrLn "'cabal.project.local' file already exists. Now overwriting it."
```

Comment:
As this looks like a logging message (and also the one below), it should go via the logging API, possibly with `notice`-log-level, rather than bypassing the logging layer and write directly to stdout.